### PR TITLE
Add PayPay payment method

### PR DESCRIPTION
**Pull request by opn-takisawa**
Allow customers to pay with PayPay, using `paypay` source type.

Sample PayPay payment method option (Light and Dark mode)

Simulator Screen Shot - iPhone 13 - 2022-10-21 at 13 49 04Simulator Screen Shot - iPhone 13 - 2022-10-21 at 13 48 39

### CODE REVIEW

**Review by hpariwat — APPROVED** [review]

**Review by kan98 — APPROVED** [review]

**Review by tookaa2 — APPROVED** [review]

File: ExampleApp/Shared/PaymentSettingTableViewController.swift [hunk 1]
```diff
@@ -85,6 +85,8 @@ class PaymentSettingTableViewController: UITableViewController {
     @IBOutlet private var myrCurrencyCell: UITableViewCell!
 
     @IBOutlet private var internetBankingBAYPaymentCell: UITableViewCell!
+    @IBOutlet private var internetBankingKTBPaymentCell: UITableViewCell!
+    @IBOutlet private var internetBankingSCBPaymentCell: UITableViewCell!
```
Comment (hpariwat) [inline ExampleApp/Shared/PaymentSettingTableViewController.swift:89]: These are unrelated changes

**Comment by rileydukeopn** [comment]
Merging has created too many problems. Moving paypay changes ONLY to a new PR: https://github.com/omise/omise-ios/pull/196
